roles/sap_ha_install_anydb_ibmdb2/tasks/platform/ascertain_platform_type.yml

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
This complete file is clearly separate role from my point of view.

This code is already used in several places in this collection.

https://github.com/search?q=repo%3Asap-linuxlab%2Fcommunity.sap_install%20ascertain_platform_type.yml&type=code

Please consider creating a role from it so there will be no more copy and paste, which is hard to support in the long term.

Reply:
Future scope outside the purview for this Ansible Role PR, the code stays as-is taken from other Ansible Roles within this Ansible Collection.